src/client/java/teammates/client/scripts/UploadBackupData.java

```diff
@@ -276,28 +276,20 @@ private static void persistProfiles(Map<String, StudentProfileAttributes> studen
     private static void adjustFeedbackResponseId(FeedbackResponseAttributes response) {
         FeedbackQuestionAttributes question = feedbackQuestionsPersisted.get(response.feedbackQuestionId);
 
-        if (feedbackQuestionIds.containsKey(question.getId())) {
-            response.feedbackQuestionId = feedbackQuestionIds.get(question.getId());
-        } else {
-            String newId = feedbackQuestionsLogic.getFeedbackQuestion(
-                    response.feedbackSessionName, response.courseId,
-                    question.questionNumber).getId();
-            response.feedbackQuestionId = newId;
-            feedbackQuestionIds.put(question.getId(), newId);
-        }
+        String newId = feedbackQuestionsLogic.getFeedbackQuestion(
+                response.feedbackSessionName, response.courseId,
+                question.questionNumber).getId();
+        feedbackQuestionIds.putIfAbsent(question.getId(), newId);
+        response.feedbackQuestionId = feedbackQuestionIds.get(question.getId());
     }
 
     private static void adjustFeedbackResponseCommentId(FeedbackResponseCommentAttributes response) {
         FeedbackQuestionAttributes question = feedbackQuestionsPersisted.get(response.feedbackQuestionId);
 
-        if (feedbackQuestionIds.containsKey(question.getId())) {
-            response.feedbackQuestionId = feedbackQuestionIds.get(question.getId());
-        } else {
-            String newId = feedbackQuestionsLogic.getFeedbackQuestion(
-                    response.feedbackSessionName, response.courseId,
-                    question.questionNumber).getId();
-            response.feedbackQuestionId = newId;
-            feedbackQuestionIds.put(question.getId(), newId);
-        }
+        String newId = feedbackQuestionsLogic.getFeedbackQuestion(
+                response.feedbackSessionName, response.courseId,
+                question.questionNumber).getId();
+        feedbackQuestionIds.putIfAbsent(question.getId(), newId);
+        response.feedbackQuestionId = feedbackQuestionIds.get(question.getId());
```

Comment:
Can revert the above two? They're not in the scope of `getOrDefault`.

Reply:
Okay done!